packages/server/scripts/duplicateUserMigration.js

```diff
@@ -1,6 +1,5 @@
-const appRoot = require('app-root-path')
-const knex = require(`${appRoot}/db/knex`)
-const roles = require(`${appRoot}/modules/core/roles.js`)
+const knex = require('@/db/knex')
+const roles = require('@/modules/core/roles.js')
```

Comment:
These server scripts could be converted to yargs commands, but up to you

Reply:
yeah i was too lazy to do it :)